charts/sequencer-relayer/values.yaml

```diff
@@ -59,3 +61,27 @@ storage:
       size: "1Gi"
       persistentVolumeName: "sequencer-relayer-storage"
       path: "/data/sequencer-relayer-data"
+
+# When deploying in a production environment should use a secret provider
+# This is configured for use with GCP, need to set own resource names
+# and keys
+secretProvider:
+  enabled: true
```

Comment:
Torn on enabling by default here, more technically correct for prod but need to have secret provider setup to use chart at all.

I think elsewhere we have disabled by default, stick to that for now?

Reply:
Good call. Thanks for catching that. It's disabled now.